.github/workflows/lint.yml

```
@@ -16,7 +16,12 @@ jobs:
         run: |
           rustup toolchain install stable
           rustup default stable
-          rustup component add rustfmt
+          rustup component add rustfmt clippy
 
+      # Formatting
       - name: Run rustfmt
         run: cargo fmt -- --check
+
+      # Linting
+      - name: Run clippy
```

```suggestion
      - name: Lint code
```

Maybe better to incorporate comment as name?